Assets/Scripts/CamControll.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CamControll : MonoBehaviour
+{
+    [SerializeField]
+    private Transform target;
+
+    [SerializeField]
+    private Vector3 tOffset;
+
+    [SerializeField]
+    private float cspeed;
+
+
+    // Update is called once per frame
+    void Update()
+    {
+        MoveCam();
+        
+    }
+
+    void MoveCam()
```

Comment:
Hai să punem și modificatorii să fie peste tot (cum deja i-ai pus la variabile) (inclusiv la Start/Update/FixedUpdate etc.).

Rider deja o să-ți sugereze treaba asta; de cele mai multe ori când Rider o să-ți sugereze ceva legat de stil, are dreptate.
<img width="530" alt="Screenshot 2022-10-30 at 12 23 28" src="https://user-images.githubusercontent.com/24657667/198873800-93c37ef5-6d57-4c4d-a79b-4607b94097c1.png">

Reply:
sorry mate but I'm a chad VSC user )))

Reply:
În cazul ăsta îți recomand să deschizi o dată proiectul în Rider, să vezi unde recomandă schimbări și unde nu, și sa încerci să-ți întipărești stilul ăla in memorie când scrii in VSC ca să fie la toți la fel codul.

Reply:
O sa trec pe Rider pentru acest proiect.